src/sage/graphs/graph_plot.py

```diff
@@ -168,8 +168,7 @@
         'Which graphviz layout program to use -- one of '
         '"circo", "dot", "fdp", "neato", or "twopi".',
     'by_component':
-        'Whether to do the spring layout by connected component -- a boolean.',
-    }
+        'Whether to do the spring layout by connected component -- a boolean.'}
 
```

Comment:
I am not sure if this is an improvement. Is there a standard regarding this?

Reply:
this is fixing a pep8 warning issued by pycodestyle, but not one already activated in our linter

Reply:
OK, then. Thanks.